shared/js/background/events.es6.js

```diff
@@ -384,7 +384,10 @@ browser.runtime.onMessage.addListener((req, sender) => {
                 return
             }
             if (req.registeredTempAutofillContentScript) {
-                return Promise.resolve({ site: { allowlisted: true, enabledFeatures: [] } })
+                // Tell autofill that it's enabled when protections are disabled
+                // TODO: This will need rethought when aligning with the
+                // `messageType` check above.
+                return Promise.resolve({ site: { allowlisted: true, enabledFeatures: ['autofill'] } })
```

Comment:
I know this is a big edge case but: This doesn't account for when we may wish to remotely disable the feature **and** the user has toggled off the protections.


Could you make a helper function for this check:
https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/82c784b4140b7ac59dab575116bd75af40a8d79a/shared/js/background/classes/site.es6.js#L122-L125

This array ideally should check all feature names for the same function.


Alternatively (and perhaps the simplest) I think we could fix the code in autofill and remove this carve out with FIXME on line 374. I can assist/fix the case in line 384 before the next release so the whole if statement can be removed.

Reply:
Updated as chatted, to remove this check all together so that autofill response falls through to getting the complete `argumentsObject` response 👍